README.md

```diff
@@ -71,6 +70,35 @@ results = collection.query(
 - __Feature-rich__: Queries, filtering, density estimation and more
 - __Free & Open Source__: Apache 2.0 Licensed
 
+## Chroma DB Server Deployment with Persistance Storage
```

Comment:
I think this is better in the `docs` than the readme. Can we remove from here and transplant to the docs?

Reply:
any specific section you recommend? How do I add these to docs?

Reply:
let's consider updating this? https://docs.trychroma.com/usage-guide#running-chroma-in-clientserver-mode

If you just remove it from this PR... I am happy to add it to docs :)